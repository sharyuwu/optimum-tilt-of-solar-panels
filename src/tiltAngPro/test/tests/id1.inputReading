id1_1
1455
665
id1_2
1455.54
665.13
