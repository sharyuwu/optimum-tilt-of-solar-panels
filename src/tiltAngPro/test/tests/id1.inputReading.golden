.inputReading
id1_1
Input 1455.0 Absolute Erros = 0.0
Input 665.0 Absolute Erros = 0.0
id1_2
Input 1455.54 Absolute Erros = 0.0
Input 665.13 Absolute Erros = 0.0
